bindings/python/generate_bindings.py

```diff
@@ -1,4 +1,82 @@
 import json
+import argparse
+import sys
+import os
+
+
+class kind:
+
+    KIND = None
+    NAME = None
+    MEMBERS = None
+    DEPTH = None
+    state_stack = []
```

Comment:
Seems like this should be `_state_stack` instead since the user should not modify this